admin_guide/manage_nodes.adoc

```diff
@@ -146,76 +156,116 @@ Pods are not scheduled for the node until the node passes the health checks that
-****
+[options="nowrap"]
+----
+$ oadm manage-node <node1> <node2> \
+    --list-pods [--pod-selector=<pod_selector>] [-o json|yaml]
+----
 
 To list all or selected pods on selected nodes:
 
-****
-`$ oadm manage-node --selector=_<node_selector>_ --list-pods [--pod-selector=_<pod_selector>_] [-o json|yaml]`
-****
+----
+$ oadm manage-node --selector=<node_selector> \
+    --list-pods [--pod-selector=<pod_selector>] [-o json|yaml]
+----
 
 == Marking Nodes as Unschedulable or Schedulable
-Use the following command to mark the nodes as unschedulable:
+[[marking-nodes-as-unschedulable-or-schedulable]]
+By default, healthy nodes with a `Ready` link:#node-conditions[status] are
+marked as schedulable, meaning that new pods are allowed for placement on the
+node. Manually marking a node as unschedulable blocks any new pods from being
+scheduled on the node. Existing pods on the node are not affected.
 
-****
-`$ oadm manage-node _<node1>_ _<node2>_ --schedulabe=false`
-****
+To mark a node or nodes as unschedulable:
 
-Marking node as unschedulable will block any new pods to be scheduled on the
-node. Existing pods on the node will not get affected.
+----
+$ oadm manage-node <node1> <node2> --schedulable=false
+----
 
-Use the following command to mark the nodes as schedulable:
+For example:
 
-****
-`$ oadm manage-node _<node1>_ _<node2>_ --schedulabe` or
+====
+[options="nowrap"]
+----
+$ oadm manage-node node1.example.com --schedulable=false
+NAME                 LABELS                                        STATUS
+node1.example.com    kubernetes.io/hostname=node1.example.com      Ready,SchedulingDisabled
+----
+====
 
-`$ oadm manage-node _<node1>_ _<node2>_ --schedulabe=true`
-****
+To mark a currently unschedulable node or nodes as schedulable:
 
-Marking node as schedulable will allow any new pods to be scheduled on the
-node.
+----
+$ oadm manage-node <node1> <node2> --schedulable
+----
 
-Instead of specifying the nodes _<node1>_ _<node2>_, you can also use
---selector=_<node_selector>_ to mark selected nodes as schedulabe or
-unschedulable.
+Alternatively, instead of specifying specific node names (e.g., `_<node1>_
+_<node2>_`), you can use the `--selector=_<node_selector>_` option to mark
+selected nodes as schedulable or unschedulable.
 
 == Evacuating Pods on Nodes
-Use the following command to evacuate all or selected pods on one or more nodes:
+Evacuating pods allows you to migrate all or selected pods from a given node or
+nodes. Nodes must first be
+link:#marking-nodes-as-unschedulable-or-schedulable[marked unschedulable] to
+perform pod evacuation.
+
+Only pods backed by a
+link:../architecture/core_objects/kubernetes_model.html#replication-controller[replication
+controller] can be evacuated; the replication controllers create new pods on
+other nodes and remove the existing pods from the specified node(s). Bare pods,
+meaning those not backed by a replication controller, are unaffected by default.
```

Comment:
@pravisankar Can you ack the above two paragraphs that have been added to / reworked?